server/middleware/blogs-permalink.js

```
@@ -0,0 +1,23 @@
+const fetchres = require('fetchres');
+const isFastFT = url => url.indexOf('NTlhNzEyMzMtZjBjZi00Y2U1LTg0ODUtZWVjNmEyYmU1NzQ2-QnJhbmRz') !== -1;
+
+module.exports = (req, res, next) => {
+	if (req.query.p && isFastFT(req.url)) {
+		const url = `http://www.ft.com/fastft/api/get_post/?id=${req.query.p}&api_key=${process.env.BLOGS_API_KEY}`;
+		return fetch(url, {
+				timeout: 500
+			})
+			.then(fetchres.json)
+			.then(data => {
+				if (data && data.status && data.status === 'ok' && data.post && data.post.url) {
+					res.redirect(302, data.post.url);
+					res.end();
+				} else {
+					next();
+				}
+			})
+			.catch(next)
```

this error happens only within the `req.query.p && isFastFT(req.url)` domain, and so, yes, I think we should allow the error to bubble up to Sentry, for you to investigate... rather than the error getting swallowed up and keeping quiet about it